### ISSUE TITLE
what is the egress gcp.credentials_json value supposed to look like?

### ISSUE THREAD

**Issue by stevenaldinger** [livekit/livekit-helm#62]
I was expecting it to be a file path with a volume mount but then saw the deployment template has no volume mounts.

From: https://github.com/livekit/livekit-helm/blob/master/egress-sample.yaml#L30-L31

Is this supposed to be raw JSON content like the following snippet? I'll happily open a PR that adds more clarity to the example if you can help me out.

```
egress:
  gcp:
    credentials_json: |
      {
        ...my JSON key contents...
      }
    bucket: my-bucket
```

**Comment by davidzhao** [comment]
Yeah, it should contain the contents of the credentials JSON that GCP gives you. Your example is the right way of embedding it here.
